src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -277,24 +277,26 @@ class MoreInfoMediaPlayer extends LitElement {
         : undefined;
     }
 
-    if (state === "idle") {
-      return supportsFeature(stateObj, SUPPORTS_PLAY)
-        ? [
-            {
-              icon: "hass:play",
-              action: "media_play",
-            },
-          ]
-        : undefined;
-    }
-
     const buttons: ControlButton[] = [];
 
-    if (supportsFeature(stateObj, SUPPORT_TURN_OFF)) {
-      buttons.push({
-        icon: "hass:power",
-        action: "turn_off",
-      });
+    if (state === "on" || state === "idle") {
```

Comment:
I'm not sure this condition is needed. Shouldn't the power off button be shown when the state is `playing`, for example?

Reply:
I though about that earlier, and my initial feeling was no, to prevent an accidental turning off if the user slightly miss-clicks but was attempting to hit "pause". Hence the condition.

However, 2 minutes ago, I thought about a related scenario. Given my point above stays (no off switch while playing visible), and we then show the off switch if paused the rules would look like this:

Off: On
On: Off
Idle: Off | Play
Playing: Pause
Pause: Off | Play

That is not ideal, since you cannot quickly/easily press pause and then play again, since the position of the buttons changes. I was wondering whether it would make sense that if a player can in general turn off, that the turn off icon is always shown (to keep the button order at the beginning consistent) but disabled if media is playing.

Reply:
Generally speaking, I think that present and disabled is better, both from a discovery standpoint (if something is disabled and has a tooltip, for example, it's easier to understand why it's disabled, vs it not showing at all), and as you said, to ensure a consistent location.
However, I'm not sure we need to protect the user here - people regularly turn media players off while they're playing, and it is currently available.

Reply:
What about always showing the "off" icon and also keeping it enabled all the time even while playing, but if pressed while playing we show a confirmation dialog to the user to prevent accidental shutoffs? Or is that also still too intrusive? 

TBH I haven't used media_players much with HA yet, so I might be unnecessarily too cautious here.

Reply:
Honestly I think this is beyond the scope of this PR.
Initially, the power off button was there in any state but "off" (in which case there was an identically looking power button to turn on).
We then had a regression, where in "idle", the power off button was not there. I think that the scope should be to revert that regression. Any change in behavior to turn off, would probably make sense as a separate PR.

BTW I just realized that for the lovelace card we only took care of idle, but not other non-off states.

Reply:
I agree with @OnFreund here, let's just address the regression